cloudformation.yaml

```diff
@@ -26,6 +27,26 @@ Parameters:
         Description: Secret see to hash IDs
         Type: String
 
+Globals:
+    Function:
+        Environment:
+            Variables:
+                EMAIL_ORIGIN: !Ref Origin
+                SALT: !Ref Salt
+                BONOBO_URL: !Ref BonoboUrl
+                BONOBO_USERS_TABLE: !Sub bonobo-${Stage}-users
+        CodeUri:
+            Bucket: content-api-dist
+            Key: !Sub ${Stack}/${Stage}/${App}-reminder/${App}.jar
+        MemorySize: 384
+        Runtime: java8
+        Timeout: 100
+        Events:
```

Comment:
`Events` cannot be used in the `Globals` section.

> [Unsupported Properties](https://github.com/awslabs/serverless-application-model/blob/develop/docs/globals.rst#unsupported-properties)
> --------
> Following properties are **not** supported in Globals section. We made the explicit call to not support them because it either made the template hard to understand or opened scope for potential security issues.
> 
> **AWS::Serverless::Function:**
> * Role
> * Policies
> * FunctionName
> * Events
> 
> **AWS::Serverless::Api:**
> * StageName
> * DefinitionBody